thirdeye/thirdeye-frontend/app/pods/components/range-pill-selectors/component.js

```diff
@@ -64,18 +68,12 @@ export default Component.extend({
       return {
         name: range.name,
         value: range.value,
-        isActive: false,
+        isActive: range.value === activeKey,
```

Comment:
nit, we could destructure `({name, value}) => {` so we can just refer to `name` and `value`

Reply:
done.